src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -141,10 +142,10 @@
  *
  * import java.io.Serializable;
  *
- * class Foo {
- *   public static final int FOO_CONST = 1;
+ * class FirstClass {
+ *   public int firstClassVariable = 1;
```

Comment:
Remove public modifiers from all lines.
Check does not care about them at all, but it will shorten example line length

Reply:
Done. Public modifiers from all lines have been removed.